### PR TITLE
Kubecon Detroit Page

### LINKED ISSUE #2865: Page for Kubecon Detroit
[See Mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=7230%3A12906)

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #2865 

- [x] Header
- [x] Our Talks
- [x] Booth
- [x] previous Kubecon
- [x] check darkmode
- [x] check spacing & font-sizes

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2864"></a>

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
[**`Preview`**](https://deploy-preview-2864--gitpod-kumquat.netlify.app/events/kubecon-detroit)


- [ ] Fix image for dark mode:
    image
- [ ] Add Title Name:
    image

File: src/routes/events/kubecon-detroit.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  import OurBooth from "$lib/components/events/our-booth.svelte";
+  import PreviousKubecons from "$lib/components/events/previous-kubecons.svelte";
+  import Explore from "$lib/components/explore.svelte";
+  import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+  data={{
+    title: "Meet Gitpod at KubeCon NA in Detroit",
+    description: "Meet Gitpod at KubeCon NA in Detroit",
```
Comment (Cahllagerfeld) [inline src/routes/events/kubecon-detroit.svelte:13]: Maybe we need to adjust this a bit more, it's just something I came up with

**Comment by chrifro** [comment]
Nice, thanks @Cahllagerfeld ✨

A few comments:

- [ ] the KubeCon logo card is too big (see mockup dimensions)
- [ ] the default line breaking for headlines shouldn't allow one word per line -> move "in" in the following line to "Detroit"

- [ ] the headline should be aligned to the left
Bildschirmfoto 2022-10-10 um 13 56 39

- [ ] the images seem blurry. You could export them in 3x from Figma and then optimize them with https://tinypng.com/

Bildschirmfoto 2022-10-10 um 13 57 42

- [ ] mobile: reduce KubeCon logo size
- [ ] mobile: reduce horizontal spacing e.g. between cards, between sub-headline text and cards, before h2 headlines
examples
Bildschirmfoto 2022-10-10 um 14 03 12
Bildschirmfoto 2022-10-10 um 14 04 46
Don't we have spacing for mobile already defined? 🤔

Afterward, it's ready to be merged from my side. 

@jacobparis in case you haven't looked at this yet, could you review the page from a messaging POV? 🙏 [[1](https://gitpod.slack.com/archives/C03Q800JKBQ/p1665048452230699?thread_ts=1664464323.032119&cid=C03Q800JKBQ)]

File: src/lib/components/events/header.svelte [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+</script>
+
+<div class="grid grid-cols-1 gap-x-small mb-x-large lg:grid-cols-2">
+  <HeaderDescription
+    class="!mb-0"
+    title="Meet us at KubeCon NA in Detroit"
+    label="Conference"
+    date="October 26-28"
+    location="Detroit, USA"
+    description=" Would be awesome to see you. Come and visit us we’ll be at Booth SU42 and also virtually."
```
Comment (jacobparis) [inline src/lib/components/events/header.svelte:13]:
```suggestion
    description="We'll be at Booth SU42. Come visit us in person or see us in our virtual booth online."
```

**Review by jacobparis — APPROVED** [review]
Copy looks great, only one phrasing recommendation

**Comment by Cahllagerfeld** [comment]
Hey @chrifro, I made the adjustments, could you re-check pls 🙏

**Comment by chrifro** [comment]
>Hey @chrifro, I made the adjustments, could you re-check pls 🙏

Better, but the general spacing still seems off and non-consistent. All spacings are defined in the [component library](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=513%3A694). We should not introduce any new spacing and stay consistent with the spacing throughout the entire website. Happy to sync on it if anything is unclear :) 

**Example:**
the top spacing should be more (see mockup). Let's use 160px just as we do on [/enterprise](https://www.gitpod.io/for/enterprise))

Bildschirmfoto 2022-10-11 um 11 51 10

The spacing between different sections varies even on the same page. Should be 160px before introducing a new section (see Figma mockup and component library)
Bildschirmfoto 2022-10-11 um 11 55 24

Bildschirmfoto 2022-10-11 um 11 55 48
Bildschirmfoto 2022-10-11 um 11 55 37

**Comment by Cahllagerfeld** [comment]
Thanks for pointing to the enterprise page, it seems like there's a component for spacing I didn't know about 🙏

**Comment by chrifro** [comment]
@pawlean is the timing correct? Or should it be specified to one day? If it's not clear yet, we can update it to " to be announced"
Bildschirmfoto 2022-10-11 um 16 15 13


Otherwise, it's ready to be merged 🙌 🎉

**Comment by pawlean** [comment]
> @pawlean is the timing correct? Or should it be specified to one day? If it's not clear yet, we can update it to " to be announced" Bildschirmfoto 2022-10-11 um 16 15 13

I'm meeting Ernst this Friday to confirm timings! Let's put TBC 🙌

Great job both of you! Thank you for getting this through, it looks SO good!

**Pull request merged**